tests/rare/ClogRemoteTlog.toml

```diff
@@ -0,0 +1,32 @@
+[configuration]
+processesPerMachine = 1
+machineCount = 20
+generateFearless = true
+minimumRegions = 2
```

Comment:
FYI: Although `generateFearless` simulates HA mode with 4 DCs, it's possible all those DCs are in the same region (it happens in this PR with rocksdb storage engine). So setting `minimumRegions` to 2 explicitly since my test relies on 2 regions.